wdio.conf.js

```diff
@@ -21,7 +21,7 @@ exports.config = {
   // then the current working directory is where your `package.json` resides, so `wdio`
   // will be called from there.
   //one file
-  specs: ["./test/specs/*.spec.js"],
+  specs: ["./test/specs/suite2.spec.js"],
```

Comment:
this means you will always execute only suite2. As I`ve commented above, better to avoid numbers in names -> maybe we can name this site a shoppingCart or checkout?